File: sandbox/ruby/src/FaxLineDeleteExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
end

fax_line_delete_request = Dropbox::Sign::FaxLineDeleteRequest.new
fax_line_delete_request.number = "[FAX_NUMBER]"

begin
    Dropbox::Sign::FaxLineApi.new.fax_line_delete(
        fax_line_delete_request,
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling FaxLineApi#fax_line_delete: #{e}"
end

File: sandbox/node/src/FaxLineDeleteExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();
apiCaller.username = "YOUR_API_KEY";

const faxLineDeleteRequest: models.FaxLineDeleteRequest = {
  number: "[FAX_NUMBER]",
};

apiCaller.faxLineDelete(
  faxLineDeleteRequest,
).catch(error => {
  console.log("Exception when calling FaxLineApi#faxLineDelete:");
  console.log(error.body);
});
